packages/data-context/package.json

```diff
@@ -27,6 +27,7 @@
     "dataloader": "^2.0.0",
     "dayjs": "^1.9.3",
     "dedent": "^0.7.0",
+    "disparity": "^3.0.0",
```

Comment:
this and `temp-dir` should be in `devDependencies` since this is test code

Reply:
I moved these from `data-context` to `launchpad` when moving the test code to a task, and also moved them into `devDependencies`, so they are no longer in `dependencies`.